src/main/java/org/breedinginsight/brapps/importer/model/imports/TermTypeTranslator.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.breedinginsight.brapps.importer.model.imports;
+
+import lombok.Getter;
+
+/**
+ Defines Ontology Term Type backend and user readable values for easy translation
+ */
+@Getter
+public enum TermTypeTranslator {
+    PHENOTYPE("PHENOTYPE", "Phenotype"),
+
+    GERM_ATTRIBUTE("GERM_ATTRIBUTE", "Germplasm Attribute"),
+
+    GERM_PASSPORT("GERM_PASSPORT", "Germplasm Passport");
+
+    public String name;
+    public String userDisplay;
+
+    TermTypeTranslator(String name, String userDisplay) {
+        this.name = name;
+        this.userDisplay = userDisplay;
+    }
+
+    public static String nameFromUserDisplay(String userDisplay){
+        for (TermTypeTranslator term: values()){
+            if (term.userDisplay.equals(userDisplay)){
+                return term.name();
+            }
+        }
+        throw new IllegalArgumentException();
+    }
```

Comment:
This can be simplified to return the enum that matches the `userDisplay` (and also remove the `name` from the enum terms)

Reply:
I ended up just making a wrapper around a map and got rid of the additional enum which I think simplified things more. If in the future the conversion will go the other way, can iterate through or use a bidirectional map.